### ISSUE TITLE
PEP8 compliance to the code

### ISSUE THREAD

**Issue by marangonico** [nofaceinbook/muxp#19]
https://www.python.org/dev/peps/pep-0008/

This is one of critical and must-have for better code readability.
I could adapt one of the modules and send it for review.

**Comment by nofaceinbook** [comment]
Yes. This would be great. Polyshing code is what I always postpone because I'm more looking for new functionalities....
